toys-core/lib/toys/loader.rb

```diff
@@ -233,7 +233,7 @@ def has_subtools?(words) # rubocop:disable Naming/PredicateName
     #
     def split_path(str)
       return str if str.is_a?(::Array)
-      @extra_delimiters ? str.split(@extra_delimiters) : [str]
+      @extra_delimiters ? str.to_s.split(@extra_delimiters) : [str]
```

Comment:
Can you call `to_s` on the str in the no-extra-delimiters case as well? (... `: [str.to_s]`)

Reply:
Done, also for Arrays.